config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -229,7 +229,7 @@ periodics:
           cpu: 6
           memory: "16Gi"
 
-- cron: '1 8 * * 1,2,3,4,5' # Run at 00:01PST (8:01 UTC) from Mon to Fri
+- cron: '1 8 * * 1,2,3,4' # Run at 00:01PST (8:01 UTC) from Mon to Thurs
```

Comment:
Can we change that to 1,2,4,5 (and run kube-dns on Wed)?

This reduces time between longest time between two consecutive core-dns tests from 3 days to 2 days, which would be actually a good thing to have.

Reply:
Done.